README.md

```diff
@@ -593,6 +595,25 @@ rosrun zivid_samples sample_capture_2d_cpp
 rosrun zivid_samples sample_capture_2d.py
 ```
 
+### Sample Capture with Settings from File
+
+This sample shows how to [configure](#configuration) the 3D capture settings
+by invoking the [load_settings_from_file](#load_settings_from_file) service and
+how to invoke the [capture](#capture) service.
```

Comment:
Maybe you can also add here that the same idea also works for 2D settings, and  describe the difference fex that you must use `load_settings_2d_from_file` service? It is also possible to have it as a separate sample or even set both 2D and 3D in the same sample (with some comment to remove it if you don't want to set 2D settings).

Reply:
I can write something about it. In the long run we want to achieve sample parity with the other languages. As such we also want to include it in the [capture tutorial](https://support.zivid.com/latest/academy/camera/capture-tutorial.html#load). It's unfortunate that the settings in ROS seem to be a state, as opposed to in Python, C++ and C#.